README.md

```diff
@@ -14,21 +14,21 @@ You can [create an issue to contact XboxDev maintainers](https://github.com/Xbox
 
 We currently provide the following services:
 
-* Discord for communication
-* XboxDevWiki to document research
-* GitHub organization to host tools and files
+* [Discord](https://discord.gg/XUDsBYD) for communication
+* [XboxDevWiki](http://xboxdevwiki.net) to document research
```

Comment:
`[XboxDevWiki.net](http://xboxdevwiki.net)`` to document research`
Maybe that would help to spreading the url so ppl will memorize it more?